src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_MediaPlayerElement.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using FluentAssertions;
+using Microsoft.UI.Xaml.Tests.Common;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Windows.Media.Playback;
+using Windows.Phone.Notification.Management;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using static Private.Infrastructure.TestServices;
+
+namespace Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Controls
+{
+	[TestClass]
+	[RunsOnUIThread]
+	public class Given_MediaPlayerElement
+	{
+		[TestMethod]
+		public void When_MediaPlayerElement_Not_SetSource_Property_Value()
+		{
+			var entity = new MediaPlayerElement();
+			entity.SetMediaPlayer(new Windows.Media.Playback.MediaPlayer());
+			entity.MediaPlayer.Play();
+			entity.GetValue(MediaPlayerElement.SourceProperty);
+
+			Assert.IsNull(entity.Source);
+		}
+
+		[TestMethod]
+		public void When_MediaPlayerElement_SetSource_Property_Value()
+		{
+			var entity = new MediaPlayerElement();
+			entity.AutoPlay = true;
+			entity.Source = Windows.Media.Core.MediaSource.CreateFromUri(new Uri("https://test-videos.co.uk/vids/bigbuckbunny/mp4/h264/720/Big_Buck_Bunny_720_10s_5MB.mp4"));
```

Comment:
upgrade this to a `const string`? or `static readonly Uri`?

Reply:
Done.